index.html

```diff
@@ -61,17 +61,17 @@
     <div class="container">
       <!-- Example row of columns -->
       <div class="row">
-        <div class="col-md-4">
+        <div class="col-lg-12 col-sm-4">

```

Comment:
Adding in a more responsive touch is a good way to go, though not having this reflected in the text sizing makes for a bit of an awkward experience.